src/components/addItems/ItemModalBackground.tsx

```diff
@@ -63,12 +94,24 @@ export default function ItemModalBackground({
       formData["Price" as keyof typeof formData] !== "" &&
       !Number.isNaN(Number(formData["Price" as keyof typeof formData]))
     ) {
+      const newRow = {
+        // get length of item table from amplify for unique id
+        id: itemLength + rows.length + 1,
```

Comment:
This would introduce bugs because it does not guarantee a unique row id. If you add two items, delete the first one, then add a third item, both items would have an id of 2. We could instead get the max id in rows then increment.